### PR TITLE
Issue 75 lxf control growth

### LINKED ISSUE #75: add lxf_control_growth.csv as dataset in the demo project
Problems arisen:

- time is not a datetime, just a offset day number. Is this what we should store in biomarker. do we need an absolute date for each dataset, then store an offset with a unit (s, m, h, d)
- biomarker names are specific, e.g. tumor volume, this could be mapped to a more generic biomarker type (e.g. volume), but would need a separate table to store maps for each dataset, e.g. tumor volume -> volume

### PR DESCRIPTION
**Pull request by martinjrobins**
Fixes #75 

- adds lxf_control_growth.csv dataset
- adds new model: BiomarkerMap, which maps from a particular column of a dataset to a generic BiomarkerType, and adds additional information specific to that dataset
- adds subject_id to Biomarker, changes time to be relative to a base datetime stored in dataset

### CODE REVIEW

File: pkpdapp/pkpdapp/models/biomarker_type.py [hunk 1]
```diff
@@ -9,18 +9,20 @@
...
     """
-    A biomarker type, for example "concentration in mg". Each
-    :model:`pkpdapp.Biomarker` is assigned on of these types.
+    A generic (not associated with a particular dataset) biomarker type, for
+    example "concentration in mg", or "volume in cm^3".
     """
 
     UNIT_CHOICES = [
-        ('mg', 'g'),
+        ('mg', 'mg'),
```
Comment (ben18785) [inline pkpdapp/pkpdapp/models/biomarker_type.py:17]: Wouldn't this make it possible to create a volume that had units g? Is it worth separating out BiomarkerMeasurements from BiomarkerUnits where there's a one to many mapping from the former to the latter? But that those mappings are hard-coded?

**Review by ben18785 — CHANGES_REQUESTED** [review]
@martinjrobins looks great, thanks. Had a few queries about the biomarker fields and whether it's worth having a separate biomarker unit field (also about the naming as am finding it a bit hard to follow what's what).

File: pkpdapp/pkpdapp/models/biomarker_map.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+from django.db import models
+from pkpdapp.models import BiomarkerType, Dataset
+
+
+class BiomarkerMap(models.Model):
+    """
+    This model contains additional information about a biomarker type within a
+    particular dataset, e.g. "Tumour volume", rather than a generic "volume"
+    biomarker type. This would correspond with a particular column in a
```
[the remark this answers is not in the document]
Comment (martinjrobins) [reply pkpdapp/pkpdapp/models/biomarker_map.py:15]: You are right for this particular, but typically a dataset would have "Tumor volume" and "Body weight" in separate columns surely? But I can delete it. In fact, I was also wondering if it is worth having both BiomarkerMap and BiomarkerType, so I might rejig this entirely

File: pkpdapp/pkpdapp/models/biomarker_type.py [hunk 3]
```diff
@@ -9,18 +9,20 @@
 
 class BiomarkerType(models.Model):
```
[the remark this answers is not in the document]
Comment (martinjrobins) [reply pkpdapp/pkpdapp/models/biomarker_type.py:10]: Yea, I was unsure about the naming as well. I'll try and improve this

File: pkpdapp/pkpdapp/models/biomarker_type.py (hunk 1 above)
Comment (martinjrobins) [reply pkpdapp/pkpdapp/models/biomarker_type.py:17]: Yea. I thinking that it might be better to include "units" as a field in BiomarkerMap, and get rid of BiomarkerType entirely, what do you think?

File: pkpdapp/pkpdapp/models/biomarker_type.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (martinjrobins) [reply pkpdapp/pkpdapp/models/biomarker_type.py:11]: ITs just "biomarker_type" now, as I've removed biomarker_map. Let me know if you have a better name. I don't think biomarker_unit covers it, as its more than just a unit, its also a description of what exactly the biomarker is measuring

**Review by ben18785 — APPROVED** [review]
Looks great @martinjrobins : think this simpler structure is much easier to comprehend and manipulate now.
